src/main/webapp/help-base64File.html

```diff
@@ -0,0 +1,20 @@
+<div>
+  <p>File parameter compatible with Pipeline, how to use it in a declarative pipeline:</p>
+  <pre>
```

Comment:
probably need `<code>` also

Reply:
`<code>`  doesn't have a good rendering for this case
<img width="909" alt="Screen Shot 2021-01-23 at 2 22 41 pm" src="https://user-images.githubusercontent.com/19728/105568622-9020c680-5d86-11eb-8064-4abc640b39d9.png">

Reply:
I meant `<pre>` _and_ `<code>`. If `<pre>` alone triggers a monospaced font then I guess that suffices.

Reply:
normally you would use pre and code together...